CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Copy and pasting the git commit messages is __NOT__ enough.
 - Added README_pypi.md to update to the general user installation PyPI instructions. See Issue #828 
 ### Changed
 - `history_vehicles_replacement_for_imitation_learning.py` now uses new Imitation action space.  See Issue #844.
+- Updated and removed some package versions to ensure that Python3.8 is supported by SMARTS. See issue #266.
```

Comment:
I assume you've tested that everything still works fine with 3.7 as well, right?

After this goes in, should we update our official stance on what versions of Python we support (for example, as we state in `README.md` and potentially elsewhere)?  If so, we should update what  we say in those places too as part of this PR.

Also, my understanding of the conversation we had about this in our last meeting was that our stance was going to change from "only version 3.7.x" to "at _least_ version 3.7".  If that's the case, do we want/need to change some of the `==` symbols in `requirements.txt` below to be `>=` as well (for Python ersions newer than 3.8)?

Reply:
Yeah there are bunch of places where the `at least` parts needs to be updated. I will modify those. I dont think `requirements.txt` requires >= , cause i have updated the packages  version based on what is supported by both `python3.7` and `python 3.8`.